### ISSUE TITLE
Not working in ExpressionEngine 3.3.0

### ISSUE THREAD

**Issue by MobsterGit** [tjdraper/custom-menu-ee#1]
Am I missing a dependent add on requirement?

**Comment by MobsterGit** [comment]
```
TypeError: e.attr(...) is undefined
http://sitstringsdesign.com/admin.php?S=0&D=cp&C=javascript&M=combo_load&ui=core,widget,mouse,position,sortable,dialog,button&plugin=ee_interact.event,ee_broadcast.event,ee_notice,ee_txtarea,tablesorter,ee_toggle_all,ee_navigation&file=json2,underscore,cp/global_start,cp/form_validation,cp/sort_helper,cp/fuzzy_filters&v=1462794468
Line 2135
```

**Comment by tjdraper** [comment]
Hi @MobsterGit, since EllisLab released the menu manager in 3.4.0, I’m not longer maintaining this add-on. https://ellislab.com/blog/entry/managing-menus-masterfully

Updating from 3.3.0 to 3.4.0 is painless and is the route I would recommend.
